elementary/monitor/api/alerts/normalized_alert.py

```diff
@@ -0,0 +1,116 @@
+import copy
+from typing import List, Optional
+
+from elementary.utils.json_utils import try_load_json
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+TABLE_FIELD = "table"
+COLUMN_FIELD = "column"
+DESCRIPTION_FIELD = "description"
+OWNERS_FIELD = "owners"
+TAGS_FIELD = "tags"
+SUBSCRIBERS_FIELD = "subscribers"
+RESULT_MESSAGE_FIELD = "result_message"
+TEST_PARAMS_FIELD = "test_parameters"
+TEST_QUERY_FIELD = "test_query"
+TEST_RESULTS_SAMPLE_FIELD = "test_results_sample"
+DEFAULT_ALERT_FIELDS = [
+    TABLE_FIELD,
+    COLUMN_FIELD,
+    DESCRIPTION_FIELD,
+    OWNERS_FIELD,
+    TAGS_FIELD,
+    SUBSCRIBERS_FIELD,
+    RESULT_MESSAGE_FIELD,
+    TEST_PARAMS_FIELD,
+    TEST_QUERY_FIELD,
+    TEST_RESULTS_SAMPLE_FIELD,
+]
+
+TEST_META_KEY = "test_meta"
+MODEL_META_KEY = "model_meta"
+ALERTS_CONFIG_KEY = "alerts_config"
+SUBSCRIBERS_KEY = "subscribers"
+CHANNEL_KEY = "channel"
+ALERT_FIELDS_KEY = "alert_fields"
+ALERT_SUPRESSION_INTERVAL_KEY = "alert_suppression_interval"
+
+
+class NormalizedAlert:
```

Comment:
Moved normalized alert from the alerts API to a different class.
Plus I have added all the alert configuration parsing to be here (alert fields used to be in somewhere else)